Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.22 AS builder
+WORKDIR /go/src/prometheus-storagebox-exporter
```

Comment:
Just FYI: The `/go/src/<name>` path structure is a relict from a time before we had Go modules. Nowadays, you can copy the code anywhere and just run "go build"